### DISCUSSION TITLE
COBOL language code conversion in Copilot

### DISCUSSION THREAD

**Discussion by rohilla007** [community/community#108970] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Does GitHub Copilot support COBOL language code conversion to other C#, C++, Java, or Python?

**Comment by gil-son** [comment, marked as answer]
In this [link](https://docs.github.com/en/enterprise-cloud@latest/get-started/learning-about-github/github-language-support#core-languages-supported-by-github-features) you can see the supported language by Github Copilot.

It may not have explicit support for COBOL, which is a programming language often used in legacy systems and enterprise environments.

If I supported you consider to mark my answer. God bless you!

**Comment by Dadzemovic** [comment]
check out https://www.cobolcopilot.com/ - it's the github copilot for Cobol
